server/handler.js

```diff
@@ -12,3 +17,5 @@ module.exports.landing = async event => {
     ),
   };
 };
+
+
```

Comment:
nit: remove newlines if not needed.

Reply:
We will eventually need to delete this file since this is a dead API

Reply:
todo: remove next sprint